evento/src/views/App/index.js

```diff
@@ -13,8 +14,9 @@ class App extends Component {
 			<div className="App">
 				<Router>
 					<Switch>
-						<Route exact path='/' component={MainPage} />
+						<Route exact path='/signin' component={SignInPage} />
 						<Route exact path='/event/:eventId' component={EventPage} />
+						<Route path='/' component={MainPage} />
```

Comment:
`<Route path='/' component={MainPage} />` does not match `/events` , which is the path for MyEvents. Maybe better way would be something like `path='/' or '/events'`.